documentation/asciidoc/topics/con_client_cert_authn.adoc

```diff
@@ -0,0 +1,19 @@
+[id='client-certificate-authentication_{context}']
+= Client certificate authentication
+
+[role="_abstract"]
+Client certificate authentication restricts in-bound connections based on the certificates that clients present.
+
+You can configure {brandname} to use trust stores with either of the following strategies:
+
+.Validate
+To validate client certificates, {brandname} requires a trust store that contains any part of the certificate chain for the signing authority, typically the root CA certificate.
+Any client that presents a certificate signed by the CA can connect to {brandname}.
+
+If you use the `Validate` strategy for verifying client certificates, you must also configure clients to provide valid {brandname} credentials.
```

Comment:
Do you think we need to explcitly state if authentication is enabled here?

Reply:
no harm. maybe users might choose validate client certs instead of authentication. that doesn't seem like a great security practice though. in any case I've updated to make that clear. thanks.